sites/updates.thunderbird.net/thunderbird/release/sep25r/index.html

```diff
@@ -0,0 +1,18 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+{#
+# This is the appeal page that will be loaded into Thunderbird directly.
+# Instead of the donation form it links to $url/donate which forces Thunderbird to open the page
+# in the user's preferred browser. This is hopefully less annoying for the end-user.
+#}
+
+{% set active_page = "appeal-sep25" %}
```

Comment:
You'll probably need to add .page-appeal-sep25 to https://github.com/thunderbird/thunderbird-website/blob/master/assets/less/appeals/jun25.less